drivers/Makefile.dep

```diff
@@ -175,6 +175,10 @@ ifneq (,$(filter hts221,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter htu21d,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_i2c
```

Comment:
```suggestion
  FEATURES_REQUIRED += periph_i2c
  USEMODULE += xtimer
```

Because there's a strong dependency on xtimer in the driver implementation.